source/MissionAction.h

```diff
@@ -55,7 +63,8 @@ class MissionAction {
 	// Determine if this MissionAction references content that is not fully defined.
 	std::string Validate() const;
 
-	const std::string &DialogText() const;
+	const DialogList &GetDialogList() const;
+	std::string MakeDialogText(const ConditionsStore &conditions, const std::map<std::string, std::string> &subs) const;
```

Comment:
Any reason not to use the previous name?
```c++
	std::string DialogText(const ConditionsStore &conditions, const std::map<std::string, std::string> &subs) const;
```

Reply:
That method has been reverted to the original.